### ISSUE TITLE
Symfony 2.1 compatibility branch needed

### ISSUE THREAD

**Issue by HenriVesala** [FabienPennequin/FPNTagBundle#10]
We need new branch for symfony 2.1 beta because there is require for symfony 2.0.x version in composer.json
In symfony 2.1 beta compatibility branch should have following line to be changed:

``` php
"symfony/framework-bundle": "2.0.*",
```

To:

``` php
"symfony/framework-bundle": "=v2.1.0-BETA1",
```

Without these changes in a new branch it is impossible to use this bundle with symfony 2.1 beta because this bundle with symfony 2.0.x requirement will make a confict in composer.

**Comment by joonsp** [comment]
Yes, this is inevitable. The sooner there's a branch for this, the better.

**Comment by FabienPennequin** [comment]
This bundle works with both versions.

Fixed with PR #11

**Comment by joonsp** [comment]
cool, thanks
